### ISSUE TITLE
Facebook add-on does not work on Android with Facebook app

### ISSUE THREAD

**Issue by fidoboy** [AppGyver/steroids#793]
The login process workflow works fine if the Facebook app is not installed, but if it is, then it doesn't work

**Comment by fidoboy** [comment]
This is still not working with 4.0.5-edge7 available today.

**Comment by fidoboy** [comment]
Still not working with 4.0.5-edge8 having the new Facebook SDK. After authorizing on Facebook app the callback is not being triggered on my app as it does without the Facebook app installed on device.

**Comment by fidoboy** [comment]
@pre What I'm using is: ```steroids.addons.facebook.login([..params...]).then(function(){...});``` as explained here: https://academy.appgyver.com/categories/16-steroids-addons/contents/134-facebook-addon-usage however, as I've described, this doesn't work on Android if Facebook app is installed; the promise is not resolved after the Facebook app is dismissed and my app comes in focus. It works fine on iOS in both cases.

**Comment by harsaharsa** [comment]
@fidoboy have you tried with the latest Facebook plugin from https://github.com/Wizcorp/phonegap-facebook-plugin ?

**Comment by fidoboy** [comment]
No, i'm using the appgyver paid addon. How can I replace with this plugin? It will the same code work?

**Comment by harsaharsa** [comment]
If you just switch out the plugin used in your app, I think the API is the same so that the Facebook Add-on code will work. It's in dire need of updating, though, so can't say for sure.

**Comment by ViliusKraujutis** [comment]
@fidoboy did you resolve your issue with free PhoneGap plugin?

**Comment by fidoboy** [comment]
No, I'm unable to find why it's not working, may be that it's necessary to configure the app hashes into the Facebook panel or something like that. I've not received any response or help from Appgyver team so any ideas are more than wellcome. It works fine however if the Facebook app is not installed on the phone, but if you have it installed, then it opens, ask you for authorization and after accepting it it returns to the app and does nothing... there is no event like in the case of Facebook dialog (with no Facebook app installed)

**Comment by ViliusKraujutis** [comment]
@fidoboy, which Appgyver guide for Facebook Add-on integration did you use?

**Comment by fidoboy** [comment]
Can't remember the location right now. The one that Appgyver provides i guess.. Same code is working fine on iOS. No problems at all. It's on Android platform where it fails
